README.md

```diff
@@ -37,7 +37,7 @@ You can download the compiled javascript directly [here](/build/postmate.min.js)
 * Child emits events that the parent can listen to.
 * Parent can `call` functions within a `child`
 * *Zero* dependencies. Provide your own polyfill or abstraction for the `Promise` API if needed.
-* Lightweight, weighing in at ~ <span class="size">`5.4kb`</span>.
+* Lightweight, weighing in at ~ <span class="size">`6.7kb`</span>.
```

Comment:
this has increased, i remeber that after my PR it actually has gone below 5kb, any reason for that?

Reply:
I was curious about the same thing. Thanks for the note.